### DISCUSSION TITLE
Angular: multiple swiper components

### DISCUSSION THREAD

**Discussion by anh-rivergon** [nolimits4web/swiper#5318] [Q&A]
Hi,

The below does not work for Angular 12x. Is it meant to work? Thanks

<body>
  <app-swiper-example></app-swiper-example>
  <app-swiper-example></app-swiper-example>
  <app-swiper-example></app-swiper-example>
</body>

**Comment by vltansky** [comment]
you didn't provide code

**Comment by anh-rivergon** [comment]
Hi, I modified the following demo code: https://codesandbox.io/s/jhxqo

I changed index.html to the following and was not able to get multiple swipers.

```
<body>
  <app-swiper-example></app-swiper-example>
  <app-swiper-example></app-swiper-example>
</body>

```

**Reply by vltansky** [reply]
try using multiple swipers inside the app component instead of duplicating app.component

**Comment by anh-rivergon** [comment]
Thanks. That put me on the right track. I was able to create a component `<carousel>` with its template having `"<swiper></swiper>"` and I could render multiple `<carousel>`.
